src/App.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import AppRoutes from './routes';
 import './App.css';
+import { CartScreen } from './components/CartScreen/cartScreen';
+
 
 const App: React.FC = () => {
   return (
     <div className="app-container">
-      <AppRoutes />
+      <CartScreen />
+      <h2>Hello World</h2>
```

Comment:
Retirar o CartScreen do` App.tsx`, Você não modifica NADA dentro do ` App.tsx`. Voltar com o AppRoutes, quando você criar uma nova tela, você precisa adicionar uma nova rota para ela e navegar por rotas.

```suggestion
      <AppRoutes />
```